src/replication.c

```diff
@@ -1418,6 +1422,16 @@ void replconfCommand(client *c) {
                 return;
             }
             c->associated_rdb_client_id = (uint64_t)client_id;
+        } else if (!strcasecmp(c->argv[j]->ptr, "pseudo-master")) {
```

Comment:
agree with @zuiderkwast , maybe a more explicit command is better, like `CLIENT PSEUDO-MASTER <on|off>`.

BTW, we should also refuse the command if server is not in `pseudo-replica` mode?